.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
@RomanGotsiy Good catch—I removed the `dist-it` dependency. That was a temporal fix for transpilling the things to ES5.